host-controller/src/main/java/org/jboss/as/host/controller/parsing/HostXml_13.java

```diff
@@ -1512,174 +1399,6 @@ private void parseSocketBindingGroup(final XMLExtendedStreamReader reader, final
-            ServerConfigResourceDefinition.GROUP.marshallAsAttribute(server, writer);
-            ServerConfigResourceDefinition.AUTO_START.marshallAsAttribute(server, writer);
-            ServerConfigResourceDefinition.UPDATE_AUTO_START_WITH_SERVER_STATUS.marshallAsAttribute(server, writer);
-            if (server.hasDefined(PATH)) {
-                writePaths(writer, server.get(PATH), false);
-            }
-            if (server.hasDefined(SYSTEM_PROPERTY)) {
-                writeProperties(writer, server.get(SYSTEM_PROPERTY), Element.SYSTEM_PROPERTIES, false);
-            }
-            if (server.hasDefined(INTERFACE)) {
-                writeInterfaces(writer, server.get(INTERFACE));
-            }
-            if (server.hasDefined(JVM)) {
-                for (final Property jvm : server.get(JVM).asPropertyList()) {
-                    JvmXml.writeJVMElement(writer, jvm.getName(), jvm.getValue());
-                    break; // TODO just write the first !?
-                }
-            }
-            if (server.hasDefined(SOCKET_BINDING_GROUP) || server.hasDefined(SOCKET_BINDING_PORT_OFFSET) || server.hasDefined(SOCKET_BINDING_DEFAULT_INTERFACE)) {
-                writer.writeStartElement(Element.SOCKET_BINDINGS.getLocalName());
-                ServerConfigResourceDefinition.SOCKET_BINDING_GROUP.marshallAsAttribute(server, writer);
-                ServerConfigResourceDefinition.SOCKET_BINDING_PORT_OFFSET.marshallAsAttribute(server, writer);
-                ServerConfigResourceDefinition.SOCKET_BINDING_DEFAULT_INTERFACE.marshallAsAttribute(server, writer);
-                writer.writeEndElement();
-            }
-            if (server.hasDefined(SSL, LOOPBACK)) {
-                ModelNode ssl = server.get(SSL, LOOPBACK);
-                writer.writeStartElement(Element.SSL.getLocalName());
-                SslLoopbackResourceDefinition.SSL_PROTOCOCOL.marshallAsAttribute(ssl, writer);
-                SslLoopbackResourceDefinition.TRUST_MANAGER_ALGORITHM.marshallAsAttribute(ssl, writer);
-                SslLoopbackResourceDefinition.TRUSTSTORE_TYPE.marshallAsAttribute(ssl, writer);
-                SslLoopbackResourceDefinition.TRUSTSTORE_PATH.marshallAsAttribute(ssl, writer);
-                SslLoopbackResourceDefinition.TRUSTSTORE_PASSWORD.marshallAsAttribute(ssl, writer);
-                writer.writeEndElement();
-            }
-
-            writer.writeEndElement();
-        }
-    }
-
-    private void writeHostProfile(final XMLExtendedStreamWriter writer, final ModelMarshallingContext context)
-            throws XMLStreamException {
-
-        final ModelNode profileNode = context.getModelNode();
-        // In case there are no subsystems defined
-        if (!profileNode.hasDefined(SUBSYSTEM)) {
-            return;
-        }
-
-        writer.writeStartElement(Element.PROFILE.getLocalName());
-        writeSubsystems(profileNode, writer, context);
-        writer.writeEndElement();
-    }
-
     /*
```

Comment:
I would also expect the following methods removed from this file:
`writeNativeManagementProtocol` https://github.com/wildfly/wildfly-core/pull/4228/files#diff-ae4aaafa96590692ce977218d56e3326R1448

`writeHttpManagementProtocol` https://github.com/wildfly/wildfly-core/pull/4228/files#diff-ae4aaafa96590692ce977218d56e3326R1467

`writeAuditLog` https://github.com/wildfly/wildfly-core/pull/4228/files#diff-ae4aaafa96590692ce977218d56e3326R1514

Reply:
fixed